heron/ckptmgr/src/java/BUILD

```diff
@@ -0,0 +1,40 @@
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+load("/tools/rules/jarjar_rules", "jarjar_binary")
+
+java_library(
+    name = "ckptmgr-java",
+    srcs = glob(
+        ["**/*.java"],
+        exclude = ["**/CheckpointManager.java"],
```

Comment:
Thanks. I missed the `java_library` and `java_binary` part the first time through.